### PR TITLE
Update GitHub app permissions screenshot

### PR DESCRIPTION
**Pull request by blampe**

### CODE REVIEW

**Comment by susanev** [comment]
@blampe do you mind updating this screenshot so its the same view our users will see instead of what we see as admins of the app? that app settings link i do not believe will be visible to folks. i am also happy to do this if you don't mind me pushing a commit.

File: themes/default/content/docs/guides/continuous-delivery/github-app.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ configure it to only be used by certain repositories.
 
 ![Configuration Page](/images/docs/github-app/org-configuration.png)
 
-The Pulumi GitHub application does not have access to your source code. It will only report
-status on pushes / pull requests that happen for repositories it is configured to access. You can
-also uninstall the GitHub application at any time without impacting your stacks or
-other Pulumi-managed resources.
+The GitHub app will report Stack operations on GitHub commits and pull requests
```
Comment (susanev) [inline themes/default/content/docs/guides/continuous-delivery/github-app.md:50]: why are we droping Pulumi from before GitHub?

Comment (blampe) [reply themes/default/content/docs/guides/continuous-delivery/github-app.md]: Typo, re-added.

**Comment by blampe** [comment]
> @blampe do you mind updating this screenshot so its the same view our users will see instead of what we see as admins of the app? that app settings link i do not believe will be visible to folks. i am also happy to do this if you don't mind me pushing a commit.

NP I took care of this.

**Review by susanev — APPROVED** [review]

**Pull request merged**